packages/blade/src/components/Radio/Radio.tsx

```diff
@@ -84,7 +86,7 @@ const _Radio: React.ForwardRefRenderFunction<BladeElementRef, RadioProps> = (
     isChecked,
     hasError,
     isDisabled: _isDisabled,
-    isRequired: groupProps.necessityIndicator === 'required',
+    isRequired: _isRequired,
```

Comment:
this might be breaking change, since existing customers who already used neccessityIndicator = required will now have to make sure to add this additional prop.

Reply:
We could avoid this by making it required if necessityIndicator is 'required' as well as controlling it with `isRequired`.